phpcs.xml.dist

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="PHP_CodeSniffer" xsi:noNamespaceSchemaLocation="phpcs.xsd">
+    <description>The coding standard for PHP_CodeSniffer itself.</description>
+    <file>source</file>
+    <arg name="colors"/>
+    <arg value="p"/>
+    <arg name="parallel" value="75" />
+    <arg name="extensions" value="php"/>
+    <rule ref="PSR2"/>
```

Comment:
I think we use PSR12 in the wrapper:
```suggestion
    <rule ref="PSR12"/>
```

Take a look into: https://github.com/OXID-eSales/coding_standards_wrapper/blob/v3.0.0/bin/oe-eshop-phpcs_with_thresholds#L62

Reply:
:+1: Nice catch, but i needed to change more. I cleaned up and force pushed :wink:

Reply:
FYI: having underscores in method names was a warning in PSR2, but it is an error in PSR12, so i had to exclude this for now from the ruleset, and let the PHP CBF cleanup again :wink: